app/Models/UserCredit.php

```diff
@@ -37,11 +37,11 @@ class UserCredit extends Model
 
     protected $guarded = [];
 
-    public function type(){
+    public function type() {
         return $this->hasOne('App\Models\UserCreditType', 'id', 'credit_type_id');
     }
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->